### PR TITLE
refactor: unify gRPC clients

### PR DESCRIPTION
**Pull request by 3Hren**

### CODE REVIEW

File: vendor/github.com/sshaman1101/grpccmd/cmd.go [hunk 1]
```diff
@@ -140,7 +140,7 @@ func dial(key *ecdsa.PrivateKey) (io.Closer, error) {
 	}
 
 	creds := util.NewTLS(TLSConfig)
-	return xgrpc.NewWalletAuthenticatedClient(ctx, creds, *remote)
+	return xgrpc.NewClient(ctx, *remote, creds)
```
Comment (sshaman1101) [inline vendor/github.com/sshaman1101/grpccmd/cmd.go]: (°.°)
(o.o) 
(O.O)
(ノಠ益ಠ)ノ彡┻━┻

Comment (3Hren) [reply vendor/github.com/sshaman1101/grpccmd/cmd.go]: ┬──┬◡ﾉ(° -°ﾉ)

Comment (sshaman1101) [reply vendor/github.com/sshaman1101/grpccmd/cmd.go]: （╯°□°）╯ ┻━┻

Comment (sshaman1101) [reply vendor/github.com/sshaman1101/grpccmd/cmd.go]: Oh, you win.

BTW I just updated my fork and [fix this](https://github.com/sshaman1101/grpccmd/commit/2b7553bdb044e0e6b0c7ebcbf18a4abbfa1dfb01). Maybe will be better to update vendor?

Comment (3Hren) [reply vendor/github.com/sshaman1101/grpccmd/cmd.go]: So your fork is currently not working? :) I think there is some kind of trouble, can't understand where exactly...

**Pull request merged**